.github/workflows/scale-test-100-gce.yaml

```diff
@@ -185,8 +184,10 @@ jobs:
           export CL2_ALLOWED_SLOW_API_CALLS=1
           export CL2_SCHEDULER_THROUGHPUT_THRESHOLD=0
 
+          gcloud compute config-ssh
```

Comment:
Could we add a comment here explaining that CL2 needs SSH access to nodes in the cluster for certain measurements to work?

Reply:
Sure, added.